introduction/getting-started.md

```diff
@@ -58,23 +58,23 @@ Investigate the source site and consider the following:
 
 - Which pages are targeted for translation? Which pages need to be excluded? Ask the client if they have a specific page list.
 
-- does the site have mixed-language content? If yes, ask the client to specify the source language(s) they need translated.
+- Does the site have mixed-language content? If yes, ask the client to specify the source language(s) they need translated.
 
-- is there an extant Translation Memory that could be used?
+- Is there an extant Translation Memory that could be used?
 
-- is there any region-specific content? Does the site use geolocation?
+- Is there any region-specific content? Does the site use geolocation?
 
-- is there any content behind a secure login?
+- Is there any content behind a secure login?
   
-- are there any subdomains? example.com and blog.example.com require two separate projects that need to be linked.
+- Are there any subdomains? Example.com and blog.example.com require two separate projects that need to be linked.
 
-- are there any other special requirements?
+- Are there any other special requirements?
 
-- is there any Javascript-generated content?
+- Is there any Javascript-generated content?
 
```

Comment:
What about other aspects? How are bots to be handled? How is new content to be handled?